ci/benchmark.sh

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env bash
+
+ZEEK_BENCHMARK_ENDPOINT="/zeek"
+
+# Setting this causes any command failures to immediately cause the script to fail.
+set -e
+
+# Don't do this for any branch that isn't from the main zeek repo.
+# TODO: is it possible to do this from cirrus.yml instead of here?
+if [ "${CIRRUS_REPO_OWNER}" != "zeek" ]; then
+    echo "Benchmarks are skipped for repositories outside of the main Zeek project"
+    exit 0
```

Comment:
Think this way is fine -- `.cirrus.yml` could do something similar at the task-level using `only_if`, but think it's ok to generally run the full CI suite on CentOS 8 and then let this additional benchmark script figure out for itself if it needs to also run (and an alternative for that check could be whether the encrypted secret env. variables exist since those are only available in commits/PRs from users that already have "write" permissions to the repo).

Reply:
This section answers your question above about whether this is run on all PRs as well, since it will only run on PRs that come directly from the zeek repo.